jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaLineMarkerTest.kt

```diff
@@ -32,8 +32,8 @@ class JavaLambdaLineMarkerTest {
 
     @Before
     fun setUp() {
-        previousShowAllGutters = SamSettings.getInstance().showAllHandlerGutterIcons
-        SamSettings.getInstance().showAllHandlerGutterIcons = true
+        previousShowAllGutters = LambdaSettings.getInstance(projectRule.project).showAllHandlerGutterIcons
```

Comment:
we can get rid of all this setup now since we're not modifying global state (I think) (And the teardown)

Reply:
Done.